packages/ui/src/components/Organisms/ContentHub.tsx

```diff
@@ -35,18 +41,16 @@ export function ContentHub({ pageSize = 10 }: { pageSize: number }) {
     ),
   });
 
-  const termsResult = useOperation(ContentHubTermsQuery, {});
```

Comment:
You should be able to keep that `useOperation` as it is, no need to prop-drill values in here. You can just inject the static query into https://github.com/AmazeeLabs/silverback-template/blob/release/apps/website/src/layouts/index.tsx like the `FrameQuery`.